src/openlcb/Defs.hxx

```diff
@@ -246,6 +246,16 @@ struct Defs
     /// station may react to this by restoring locomotive speed settings.
     static constexpr uint64_t CLEAR_EMERGENCY_STOP_EVENT = 0x010000000000FFFCULL;
 
+    /// "Power supply brownout detected below minimum required by node"
```

Comment:
Isn't there a second of this with a similar flavor but different formulation? It might be worth adding both of them.

Reply:
Yes, there is a "below standard" one as well. I wasn't sure if it should be added or not but can add it.